hack/release.toml

```diff
@@ -155,6 +155,22 @@ machine:
     servers:
       - /dev/ptp0
 ```
+"""
+
+    [notes.watchdog]
+        title = "Hardware Watchdog Timers"
```

Comment:
i guess we also mention this is  only available for amd64?

Reply:
It is available on aarch64. We actually had all drivers built-in previously, so I didn't have to modify aarch64 config except for sysfs

Reply:
oh, were they built in? I guess we need to keep them also as modules if not, just to be in parity with amd64 :thinking: . I'm not so sure, @smira wdyt?

Reply:
https://github.com/siderolabs/pkgs/blob/6364d991db023e5f614cbbc450b4afd471e16203/kernel/build/config-arm64#L4441

Maybe some of those mustn't be modules due to the way they need to be probed, so that should be checked

Reply:
it supports PCI id database https://cateee.net/lkddb/web-lkddb/WATCHDOG.html, but we might hit other issue like we try to read before udev picks them up, I was more talking about these: 

```text
❯ grep IPMI_WA
kernel/build/config-arm64
3586:# CONFIG_IPMI_WATCHDOG is not set

kernel/build/config-amd64
3157:CONFIG_IPMI_WATCHDOG=m

❯ grep MFD_CORE
kernel/build/config-arm64
4514:CONFIG_MFD_CORE=y

kernel/build/config-amd64
3630:CONFIG_MFD_CORE=m

❯ grep SP5100
kernel/build/config-amd64
3581:CONFIG_SP5100_TCO=m
```

which are not consistent across amd64 and arm64

Reply:
we can fix it up, but it shouldn't be related to to this PR specifically. 

I think arm64 in general needs a module cleanup the way I did for amd64 - try to make everything which can be a module to be a module.

arm64 kernel is not compressed, so moving to initramfs.xz is a good thing imho

Reply:
yeh, arm64 needs some cleanup

Reply:
SP5100 is AMD WDT, MFD is for Intel one, but IPMI might work for ARM. Regarding PCI ID DB: aarch64 WDTs are typically part of the SoC